backend/src/main/java/com/woowacourse/levellog/common/exception/LevellogException.java

```diff
@@ -8,9 +8,17 @@
 public abstract class LevellogException extends RuntimeException {
 
     private final HttpStatus httpStatus;
+    private final String clientMessage;
 
     protected LevellogException(final String message, final HttpStatus httpStatus) {
         super(message);
         this.httpStatus = httpStatus;
+        this.clientMessage = null;
+    }
+
+    protected LevellogException(final String message, final HttpStatus httpStatus, final String clientMessage) {
```

Comment:
개인적으로 message를 한 곳에 모는 것을 선호합니다.
```suggestion
    protected LevellogException(final HttpStatus httpStatus, final String message, final String clientMessage) {
```